src/components/ens-registration/RegistrationReviewRows/RegistrationReviewRows.tsx

```diff
@@ -11,17 +11,33 @@ import {
   Text,
 } from '@rainbow-me/design-system';
 import { REGISTRATION_MODES } from '@rainbow-me/helpers/ens';
+import { useInterval } from '@rainbow-me/hooks';
+
+const MIN_LONG_PRESS_DURATION = 200;
+const LONG_PRESS_INTERVAL = 100;
```

Comment:
we have 69 in `FeesGweiInput` maybe we should use the same interval

Reply:
sure